src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -342,6 +342,9 @@
   <data name="IDS_FeatureExceptionFilter" xml:space="preserve">
     <value>exception filter</value>
   </data>
+  <data name="IDS_FeatureGuardStatements" xml:space="preserve">
+    <value>guard statements</value>
```

Comment:
Probably will be perceived as too jargony.  Perhaps "negated-condition if statements"

Reply:
Interesting.  I actually figured we would call these 'guard statements'.  So it fit as a general term.  Especially if we wan to support this for `while !(...)` and `do ... while !(...)`.  Maybe `negated conditional statements`?

Reply:
Perhaps "negated-condition statements"? The ```negated conditional statements``` somehow implies the statements are negated in some way.

---
In reply to: [238454812](https://github.com/dotnet/roslyn/pull/30945#discussion_r238454812) [](ancestors = 238454812)

Reply:
That works for me!